CMakeLists.txt

```diff
@@ -125,7 +125,8 @@ hpx_info("HPX version: ${HPX_VERSION}")
 unset(HPX_COMPONENTS CACHE)
 unset(HPX_EXPORT_TARGETS CACHE)
 unset(HPX_EXPORT_INTERNAL_TARGETS CACHE)
-unset(HPX_LIBS CACHE)
+unset(HPX_ENABLED_LIBS CACHE)
+unset(HPX_ENABLED_MAIN_LIBS CACHE)
```

Comment:
`HPX_ENABLED_CORE_LIBS`?

Reply:
This falls under "check consistency". It's not actually for `hpx_core`, it's for the non-`hpx_core` part, and on top of that the variable may not be needed anymore (the `add_hpx_module` at least in the current iteration adds the modules to the correct library).